### DISCUSSION TITLE
Seeking Assistance for Flagged GitHub Accounts

### DISCUSSION THREAD

**Discussion by sarikarai29** [community/community#112247] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Dear GitHub Community,

I trust this message finds you well. I am writing to seek urgent assistance regarding the flagging of my GitHub accounts - [https://github.com/pushpakrai1607](https://github.com/pushpakrai1607) and [https://github.com/pushpakrai](https://github.com/pushpakrai). Unfortunately, both accounts have been flagged, and I am currently facing challenges in understanding the reasons behind this action.

I am eager to address any violations or misunderstandings that may have led to the flagging of my accounts and work towards resolving this issue promptly. Your guidance and support in clarifying the situation and reinstating my accounts to their normal status would be immensely appreciated.

Could you please provide me with information on why these accounts were flagged and the necessary steps I can take to rectify any issues? Restoring access to my accounts is crucial for me to continue my contributions effectively on the platform.

Thank you in advance for your attention to this matter. I look forward to your prompt assistance in resolving this issue and restoring professionalism to my GitHub accounts.

![second](https://github.com/community/community/assets/157900528/8b1ca249-f904-457e-88be-7a9e8203c6cc)

![sssss](https://github.com/community/community/assets/157900528/39431580-fc32-4a8a-8879-ef2155c71268)

**Comment by queenofcorgis** [comment, marked as answer]
Hi @sarikarai29 ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). The community is unable to assist with nor escalate with your situation. 

Please note: tickets will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
